string moveType #运动方式,跟原youbot控制器一样 当使用Goto时自动行进
float64 speed #运动速度
#当且仅当使用Goto模式时有用
float64 x
float64 z
float64 alphe

